migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/MisoServiceManager.java

```diff
@@ -922,7 +925,6 @@ public void setDefaultBoxDao() {
 
   private void updateBoxDaoDependencies() {
     if (poolDao != null) poolDao.setBoxStore(boxDao);
-    if (requestManager != null) requestManager.setBoxStore(boxDao);
```

Comment:
And then similar lines in `updateXDependencies` where X = AuthorizationManager, ChangeLogDao, SecurityProfileStore.